### PR TITLE
Splitting quantize_tensor and quantize_input

### PR DESCRIPTION
**Pull request by jchen351**
**Description**: Describe your changes.
Splitting quantize_tensor and quantize_input into quantize_activation_tensor, quantize_weight_tensor, quantize_activation and quantize_weight
**Motivation and Context**
This will reduce the naming confusion

### CODE REVIEW

File: onnxruntime/python/tools/quantization/onnx_quantizer.py [hunk 1]
```diff
@@ -723,7 +723,21 @@
             or (tensor_name in self.generated_value_names)
         )
```
Comment (chenfucn) [inline onnxruntime/python/tools/quantization/onnx_quantizer.py:725]: >False, False, False [](http://example.com/codeflow?start=53&length=19)

Three 'False' in a roll, a little hard to read, consider:
initializer_use_weight_qType=False, reduce_range=False, op_level_per_channel=False

File: onnxruntime/python/tools/quantization/onnx_quantizer.py [hunk 2]
```diff
@@ -723,7 +723,21 @@
...
 
-    def quantize_inputs(
+    def quantize_activation(self, node, indices, from_subgraph=False):
```
Comment (chenfucn) [inline onnxruntime/python/tools/quantization/onnx_quantizer.py:726]: >quantize_weight [](http://example.com/codeflow?start=8&length=15)

Consider adding comments explaining: in what circumstances a weight is not an intializer?

**Comment by chenfucn** [comment]
>     def is_input_a_weight(self, input_name):

Change this to is_input_an_initializer to be consistent with rest of the name change?

---
Refers to: onnxruntime/python/tools/quantization/onnx_quantizer.py:292 in 5ab6073. [](commit_id = 5ab6073ec168d25c40199d759a9585209865aa93, deletion_comment = False)

**Comment by chenfucn** [comment]
It appears to me that we used to use 'initializer' as a synonym for 'weight', and this change is aim at changing that? It would be great if we can have explanation of this in comments.

**Review by yufenglee — APPROVED** [review]
:shipit:

**Pull request merged**